websites/G/Gamepedia/presence.ts

```diff
@@ -1,266 +1,281 @@
-const presence = new Presence({
-  clientId: "652880245371699222"
-});
+if (
+  !(
+    (
+      document.location.hostname.endsWith("fandom.com") &&
+      document.location.pathname.includes("/wiki/") &&
+      !document.querySelector("#netbar")
+    )
+    // Only run on Gamepedia wikis.
+  ) &&
+  !(document.location.hostname === "www.fandom.com")
```

Comment:
![image](https://user-images.githubusercontent.com/9348108/114399634-6dda6000-9ba1-11eb-90ba-ee3c66090f0a.png)
also fix deepscan yeah didn't notice it.